types/listeningaddr.pp

```diff
@@ -1 +1 @@
-type Simp_snmpd::Listeningaddr = Tuple[Simp_snmpd::Protocol, Variant[Simplib::IP::V4, Simplib::IP::V6::Bracketed, Simplib::Hostname], Simplib::Port]
+type Simp_snmpd::Listeningaddr = Tuple[Simp_snmpd::Protocol, Variant[Simplib::IP::V4, Simplib::IP::V6::Bracketed, Simplib::Hostname, Stdlib::Unixpath], Optional[Variant[Simplib::Port,String]]]
```

Comment:
What is the purpose of the Optional[Variant[Simplib::Port,String]] ?
Do you have a test for this type?